CMakeLists.txt

```diff
@@ -133,6 +133,8 @@ message(STATUS "QUDA target: ${QUDA_TARGET_TYPE}")
 # ######################################################################################################################
 
 # build options
+option(QUDA_LARGE_KERNEL_ARG "enable large kernel arg support" OFF)
```

Comment:
I think this should be marked as `advanced` --- with my main motivation more being that I'd like to avoid folks "accidentally" turning it on.

Reply:
[9f897ac](https://github.com/lattice/quda/pull/1364/commits/9f897ac922ab8f846d3c52a1df6b09cd6ae314f1)